spec/models/dialog_serializer_spec.rb

```diff
@@ -24,7 +24,7 @@
       {
         "description"  => description,
         "buttons"      => buttons,
-        "label"        => label,
+        "label"        => name,
```

Comment:
Wouldn't this "label" also change to "name"?

Reply:
It will eventually but that requires the migration which isn't merged yet.

Reply:
Do you have a link to the migration PR?

Reply:
https://github.com/ManageIQ/manageiq-schema/pull/142